src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  width: 100px;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
you should add styles for all possible modifiers (from stars--1 to stars--5), just copy them from the stars task